runtime/compiler/aarch64/codegen/ARM64PrivateLinkage.cpp

```diff
@@ -413,7 +413,7 @@ void TR::ARM64PrivateLinkage::createPrologue(TR::Instruction *cursor)
    TR::RealRegister *vmThread = machine->getRealRegister(properties.getMethodMetaDataRegister());   // x19
    TR::RealRegister *javaSP = machine->getRealRegister(properties.getStackPointerRegister());       // x20
 
-   setInterpretedMethodEntryPoint(cursor);
+   TR::Instruction *entryPointInstruction1 = cursor;
```

Comment:
Why not call this `beforeInterpreterMethodEntryPointInstruction` to make it a bit clearer?